lib/PersistHelper.ts

```diff
@@ -1001,3 +1019,181 @@ export const retrieveLoginMessageSentStatus = async ({
 
     return result[0].isLoginMessageSent;
 };
+
+type LastBridgedMessageInfo = {
```

Comment:
for a 1000+ lines file please avoid defining types here, its better to import them